apps/src/storage/levelbuilder/ManifestEditor.jsx

```diff
@@ -39,6 +72,11 @@ class ManifestEditor extends React.Component {
     );
     return (
       <div>
+        {this.state.notice && (
```

Comment:
right now, `this.state.notice` will always be true because the default/empty value is '' instead of `null`